### DISCUSSION TITLE
How the Add Cursor at Position works?

### DISCUSSION THREAD

**Discussion by augustocdias** [mg979/vim-visual-multi#162] [Q&A] [answered]
So the way I imagine is I press `\\\` and it will add a cursor at a position. I go to another place in my text, press `\\\` again and it will add another cursor at the position of the cursor.

What is happening is that it is entering and leaving the VM mode when I press `\\\`. Am I understanding it wrong?

I also tried to remap the `C-Arrows` keys to `M-Arrows` to not conflict with MacOs own shortcuts and I'm getting errors when loading nvim. Is this the correct way to do it?

```vim
let g:VM_maps["Add Cursor Down"]             = '<M-Down>'
let g:VM_maps["Add Cursor Up"]               = '<M-Up>'
```

**Comment by mg979** [comment, marked as answer]
I tested briefly those mappings in nvim and they work. It's probably some problem in your configuration or Mac-specific and I can't help you fix those, I don't have a Mac anyway.

The `\\\` mapping works better with the mouse, it's awkward to use without it because you should keep mappings toggled off or you'd move cursors with motions. I never use this functionality tbh.
